debian/k8s-cni/99_bridge.conf

```diff
@@ -0,0 +1,18 @@
+{
+  "cniVersion": "0.1.0",
+  "name": "kubenet",
```

Comment:
Great! Then my obvious second question is, why don't we use kubenet in ALL deployments if it works everywhere? :)

No more flannel/weave dep then, unless they provide some really important side effect of course.